src/System.Private.CoreLib/src/System/Threading/SpinLock.cs

```diff
@@ -305,11 +294,6 @@ private void ContinueTryEnter(int millisecondsTimeout, ref bool lockTaken)
                 startTime = TimeoutHelper.GetTime();
             }
 
-            if (SpinLockTrace.Enabled)
-            {
```

Comment:
@jkotas  This is the only extra change from Anipik/coreclr@03bf95c#diff-ad4906f6671b58df9f8f4063744b0af3 . Do I need to add this again with ifdef or we are good without it also?

Reply:
I do not think you need to worry about it. It should be fine to delete it here since it is not in CoreCLR.